src/language-js/print/array.js

```diff
@@ -125,4 +136,32 @@ function printArrayItems(path, options, printPath, print) {
   return printedElements;
 }
 
+function printArrayItemsUsingFill(path, options, printPath, print) {
+  const parts = [];
+
+  path.each((childPath, i, elements) => {
+    const isLast = i === elements.length - 1;
+
+    parts.push([print(childPath), isLast ? "" : ","]);
+
+    if (!isLast) {
```

Comment:
nit: instead of burying the main
part of the function in the if
body, why not an early return

seomthing liek:

`if (isLast) return undefeind;`

Reply:
It's not really the main part of the function, so I think it's okay.